scripts/lib/install

```diff
@@ -104,6 +104,9 @@ fi
 # Check for a supported OS release.
 apt-get install -y lsb-release sudo
 os_info="$(lsb_release --short --id --release --codename)"
+if [ "$os_distro" == "rhel" ]; then
+    os_info="$(echo $os_info | sed -r 's/^(\S+)\s+(\S+)\s+\S+$/\1\n\2\n\2/')"
```

Comment:
Can you add a comment explaining what values this is mapping between?  Seems like good documentation for a sed line.